assets/js/index.js

```diff
@@ -7,6 +8,7 @@ require('bootstrap');
 require('select2');
 
 $(() => {
+	base();
```

Comment:
hehe. This is the develop branch which uses Babel/Browserify. 
Should land in 3.0 hopefully this week.